ci/prow/e2e-crd-variable

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Deploy KMMO..."
+make deploy
+
+echo "Wait until the KMMO Deployment is Available and "
+echo "Describe the Deployment and get its YAML if that failed..."
```

Comment:
I think we can remove that comment.

Reply:
Why? It is pretty informative about what next lines will do.

Reply:
I think that the comments should describe the happy flow, in case of an error we just print some debugging info but there is no reason to put it there to begin with.

"Too much documentation" isn't always good IMO.

But if you insist, we can keep it.

Reply:
We could put the `kubectl describe` in the troubleshooting step. If we collect that output at the end, we can remove the operands and the comment.

Reply:
It looks like a good solution. WDYT @ybettan ?